### DISCUSSION TITLE
Remove the default account creation.

### DISCUSSION THREAD

**Discussion by pangolp** [azerothcore/azerothcore-wotlk#4264] [Ideas]
Hello good afternoon, the reason for this topic, is that I would like to know what you think, about removing the creation of the default accounts, and move them instead, as optional scripts. The reason of this order, is that whenever I make an installation of AzerothCore, I remove those accounts and I find it interesting, if they could be in an optional way.

I understand, that who proposed the creation of default accounts, thought about it, inspired by the fact that those who compiled the emulator, had practically everything within reach, but I think that it would not be bad, to analyze again if it is really necessary that they are created.

Without further reason, I look forward to your ideas.
Greetings.

**Comment by Kitzunu** [comment]
Yes. Please.

**Comment by Malow** [comment]
Agreed. Maybe the DB-assembler could have an option to create them for those who want them?

**Reply by Kitzunu** [reply]
Sure, like a specific option to install accounts

**Reply by pangolp** [reply]
I liked the idea.

**Reply by Helias** [reply]
I agree too :+1:

**Reply by pangolp** [reply]
I know that maybe this is another topic of discussion, but could it be possible to create an assembler with python?

**Comment by pangolp** [comment]
This can be closed, since the change was made a long time ago.
